plugins/generator-1.14.4/forge-1.14.4/procedures/entity_get_shootpower.java.ftl

```diff
@@ -0,0 +1 @@
+(((projectileentity.getEntity() instanceof ${generator.map(field$projectiles, "projectiles")}) && (projectileentity != null))?((${generator.map(field$projectiles, "projectiles")}) projectileentity).getMotion().distanceTo(Vec3d.ZERO):0)
```

Comment:
the null check is after the usage. This may throw an npe

Reply:
It's done. I changed the first check with `&&` in two checks with the `!= null` check first.